apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/Google.Cloud.Spanner.Data.IntegrationTests.csproj

```diff
@@ -23,5 +23,10 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
```

Comment:
I think we should move the MySQL tests out into a different project at some point... they're not integration tests for Spanner. (Unless we want to run the stress tests as part of normal integration testing, moving *all* of them into a new project would probably make sense. That can be done later though, and we may have already discussed it.)